tools/interop-v2-reference.py

```
@@ -0,0 +1,842 @@
+    lz4_cv = by_name["lz4_roundtrip_compressible"]
+    _require(
+        len(bytes.fromhex(lz4_cv["payload_hex"])) < lz4_cv["original_size"],
+        "the 'compressible' vector did not compress — vector loses its point",
+    )
+
+    # Every structural reject vector must raise.
+    for rv in built["reject_vectors"]:
+        try:
+            decode_container(bytes.fromhex(rv["container_hex"]))
+        except V2Error:
+            pass
```

Ruff (0.16.2)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)